jabref.install4j

```diff
@@ -750,4 +750,9 @@ return true;</string>
   </mediaSets>
   <buildIds buildAll="true" />
   <buildOptions verbose="false" faster="false" disableSigning="false" disableJreBundling="false" debug="false" />
+  <jvmArguments>
```

Comment:
Note that if these settings go missing, it may caused by a plain save of the config using the Install4J GUI. I'm trying to add it as "VM options file" at https://github.com/JabRef/jabref/commit/53ffd02287a0cd4b83115243f631df6551d08086, which seems to work.